packages/core/src/blocks/TableBlockContent/TableBlockContent.ts

```diff
@@ -42,6 +45,25 @@ export const TableBlockContent = createStronglyTypedTiptapNode({
     );
   },
 
+  // Since we're using ProseMirror's default table column resizing plugin, we
```

Comment:
I think this is a bit hacky, as it won't hide the resize options for non-editable users. I'd rather ship this PR without this fix (i.e.: only fix multi-column) than with a hacky fix.

For a proper solution, I'd investigate whether we can disable the relevant prosemirror plugins (i.e.: columnResizing) when the editor is not editable. I'm not sure it's trivial to do this though when switching from editable to non-editable and back, as I'm not sure we can easily disable / enable plugins at that point